packages/components/src/components/DistributionChart.tsx

```diff
@@ -75,10 +102,33 @@ export const DistributionChart: React.FC<DistributionChartProps> = (props) => {
       );
       widthProp = 20;
     }
-    const domain = shapes.value.flatMap((shape) =>
+    const predomain = shapes.value.flatMap((shape) =>
       shape.discrete.concat(shape.continuous)
     );
 
+    const domain =
+      xAxis === "dateTime"
+        ? predomain.map((p) => ({ dateTime: p.x, y: p.y }))
+        : predomain;
+
+    const data =
+      xAxis === "dateTime"
+        ? shapes.value.map((val) => {
+            return {
+              ...val,
+              continuous: val.continuous.map((p) => {
+                return { dateTime: p.x, y: p.y };
+              }),
+              discrete: val.discrete.map((p) => {
+                return { dateTime: p.x, y: p.y };
+              }),
+              samples: val.samples.map((p) => {
+                return { dateTime: p, y: 0 };
```

Comment:
Does this y: 0 even get used? Any reason this isn't just `samples: val.samples`?